src/StripeTests/Entities/StripeInvoiceLineItemTest.cs

```diff
@@ -0,0 +1,53 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeInvoiceLineItemTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            // TODO: Figure out if we should test it this way as the class
+            // is generic for invoice items and line items.
```

Comment:
Flagging this one as stripe-dotnet has one unified class for 2 separate resources which makes testing it weird.

Reply:
IMO that's something we should fix as it's inconsistent with the API and every other library, but in the meantime this is fine.